app/ts/PreferencesTab/Elicitation/MatchingElicitation/MatchingSetImportance/MatchingSlider/MatchingSlider.tsx

```diff
@@ -27,13 +26,15 @@ export default function MatchingSlider({
     mostImportantCriterionId,
     setPreference
   } = useContext(ElicitationContext);
-  const {criteria, pvfs} = useContext(PreferencesContext);
+  const {getCriterion, pvfs} = useContext(PreferencesContext);
 
-  const mostImportantCriterion = criteria[mostImportantCriterionId];
+  const mostImportantCriterion = getCriterion(mostImportantCriterionId);
```

Comment:
We are doing this several times, we might as well consider storing and exposing the entire criterion instead of just the id.